packages/sdk/src/types/json.ts

```diff
@@ -291,41 +202,16 @@ function transformConcordiumTypes(obj: unknown, replacer?: ReplacerFun): any {
  *
  * @throws {JsonCircularReferenceError} If a circular reference is found.
  */
-export function jsonStringify(
-    value: any,
-    replacer?: ReplacerFun,
-    space?: string | number
-): string;
-
-/**
- * Stringify, which ensures concordium domain types are stringified in a restorable fashion.
- *
- * @param value A JavaScript value, usually an object or array, to be converted.
- * @param replacer An array of strings and numbers that acts as an approved list for selecting the object properties that will be stringified.
- * @param space Adds indentation, white space, and line break characters to the return-value JSON text to make it easier to read.
- *
- * @throws {JsonCircularReferenceError} If a circular reference is found.
- */
-export function jsonStringify(
-    value: any,
-    replacer?: (number | string)[] | null,
-    space?: string | number
-): string;
 export function jsonStringify(
     input: any,
-    replacer?: any,
+    replacer?: ReplacerFun,
     space?: string | number
 ): string {
-    // Runs replace function for concordium types prior to JSON.stringify, as otherwise
-    // an attempt to run `toJSON` on objects is done before any replacer function.
-    const transformed = transformConcordiumTypes(
-        input,
-        typeof replacer === 'function' ? replacer : undefined
-    );
-    return JSON.stringify(
-        transformed,
-        // Only add replacer if it hasn't already been run in the concordium transformer function.
-        typeof replacer === 'function' ? undefined : replacer,
-        space
-    );
+    const checkCircular = getCheckCircular();
```

Comment:
No need to check for circular references anymore, since `JSON.stringify` will take care of that now